source/puppet/4.4/reference/config_important_settings.markdown

```diff
@@ -171,6 +172,8 @@ Puppet Server has [its own configuration files][puppetserver_config_files]; cons
 * [`jruby-puppet`][jruby_puppet] --- Provides details on tuning JRuby for better performance.
 * [`JAVA_ARGS`][jvm_heap_config] --- Instructions on tuning the Puppet Server memory allocation.
 
+Also, Puppet Server uses its own [auth.conf][] file by default instead of its `puppet-admin` section to authorize client requests.
```

Comment:
That was my first idea, but I changed my mind since the deprecated method is still allowed. If we're cool with removing the deprecated stuff, that works for me. :+1: